backend/database/sql/createtables.sql

```diff
@@ -34,6 +27,16 @@ CREATE TABLE Courses (
     FOREIGN KEY (subject_code) REFERENCES Subjects(subject_code)
 );
 
+CREATE TABLE Terms (
+    term_id INT,
+    course_code VARCHAR(10) , 
+    start_date DATE,
+    end_date DATE,
+    term_season VARCHAR(10)
```

Comment:
missing comma here, should be "term_season VARCHAR(10),"

Reply:
also in the first table last row there's an extra comma could you also delete that, thanks